lib/gitlab/client/users.rb

```diff
@@ -50,9 +50,9 @@ def user(id=nil)
     def create_user(*args)
       options = Hash === args.last ? args.pop : {}
       body = if args[2]
-               { email: args[0], password: args[1], username: args[2] }
+               { email: args[0], password: args[1], username: args[2], name: args[0] }
              else
-               { email: args[0], password: args[1], name: args[0] }
+               raise ArgumentError.new("Missing required parameters")
              end
```

Comment:
Can we do `body = { email: args[0], password: args[1], username: args[2], name: args[0] }` instead? 🤔